### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Omkar-Ghate**
For the home page component. Please review the code and design.

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
Looks good for me 👍 code is readable and well structured

File: pages/homepage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
+.hero-image {
+    margin: 8px auto;
+	padding: 1.5rem;
+    min-height: 26rem;
+    background-image: url(../../assets/banner-img.jpg);
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    color: white;
```
Comment (zoef-yt) [inline pages/homepage/homepage.css]: use var

File: pages/homepage/homepage.html [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+            </div>
+        </div>
+
+        <div class="spacer-3rem"></div>
+        <div class="spacer-3rem"></div>
+
+        <div class='homepage-container'>
+
+            <h2 class='text-center feature-title'>Featured Products</h2>
+            <div class='title-underline'></div>
```
Comment (zoef-yt) [inline pages/homepage/homepage.html]: > `<hr> `

tag can be used here just a suggestion for semantic.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: pages/homepage/homepage.css (hunk 1 above)
Comment (Omkar-Ghate) [reply pages/homepage/homepage.css]: even for smaller use cases?

File: pages/homepage/homepage.html (hunk 2 above)
Comment (Omkar-Ghate) [reply pages/homepage/homepage.html]: to wrap the featured product's title?

File: pages/homepage/homepage.css (hunk 1 above)
Comment (zoef-yt) [reply pages/homepage/homepage.css]: mostly yes

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Pull request merged**